### ISSUE TITLE
Does database replication require port-forwarding?

### ISSUE THREAD

**Issue by picrypto** [orbitdb/orbit-db#442]
Do nodes need to have a port forwarded to accept incoming connections for orbit-eb database replication to occur between them? Could two nodes replicate their databases between each other if both of them didn't have port forwarding enabled and were on different networks?

**Comment by lukas2005** [comment]
Yes nodes do need to have port-forwarding for communication unless you will tell ipfs to use an external signal server like in #468

**Comment by aphelionz** [comment]
Moving this to the Field Manual to go into more detail
